### PR TITLE
[Collect] Fixes default sort and copy changes

### PR DESCRIPTION
**Pull request by l2succes**
- https://artsyproduct.atlassian.net/browse/GROW-925
- https://artsyproduct.atlassian.net/browse/GROW-928

### CODE REVIEW

File: src/Apps/Collect/__tests__/routes.test.tsx [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+      route = find(routes, ["path", "/collect/:medium?"])
+    })
+
+    it("sends correct default variables for query", () => {
+      expect(route.prepareVariables(params, {})).toEqual({
+        sort: "-decayed_merch",
+      })
+    })
+
+    xit("renders", async () => {
```
Comment (l2succes) [inline src/Apps/Collect/__tests__/routes.test.tsx]: still working on getting this test to work. It's a little tricky as it's attempting to test the router as well 
but I may just drop down to testing only the `CollectApp` and the relay container instead.

To get the fixes in, I'm commenting it out for now but will fix it in another PR

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit bf4a4e0ac8ca26a67b20e9a1fa73521e88d137f6

https://deploy-preview-1387--artsy-reaction.netlify.com

File: src/Apps/Collect/Components/ArtworkGrid/index.tsx [hunk 2]
```diff
@@ -166,39 +166,43 @@ class Filter extends Component<Props, State> {
   }
 
   renderFilters({ user, filters, mediator, hideTopBorder }) {
-    const hasLabFeature =
-      user &&
-      user.lab_features &&
-      user.lab_features.includes("New Buy Now Flow")
-    const enableBuyNowFlow = sd.ENABLE_NEW_BUY_NOW_FLOW || hasLabFeature
+    try {
+      const hasLabFeature =
```
Comment (eessex) [inline src/Apps/Collect/Components/ArtworkGrid/index.tsx]: Where is the error coming from here?  Not used to seeing the `try`/`catch` syntax on a function that is not declared `async` or using `await`.

File: src/Apps/Collect/__tests__/routes.test.tsx [hunk 3]
```diff
@@ -0,0 +1,170 @@
+import { mount } from "enzyme"
+import { Resolver } from "found-relay"
+import createRender from "found/lib/createRender"
+import getFarceResult from "found/lib/server/getFarceResult"
```
Comment (eessex) [inline src/Apps/Collect/__tests__/routes.test.tsx]: Would think this unused import would throw an error -- maybe worth commenting out as well, or removing the comments and replacing with a TODO until implementation is fixed.

**Comment by l2succes** [comment]
merge on green

**Pull request merged**